packages/core/test/lib/request.test.ts

```diff
@@ -202,4 +206,25 @@ describe('sessionToSentryRequest', () => {
       }),
     );
   });
+
+  it('uses tunnel as the url if it is configured', () => {
+    api = new API(ingestDsn, {}, tunnel);
```

Comment:
Can we do a test that asserts the opposite? (That it doesn’t use the tunnel)

Reply:
Done